README.md

```diff
@@ -166,7 +156,7 @@ To get a local copy up and running, follow these steps.
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/XRichardroidX/Add_Books_to_Cart/issues).
+Feel free to check the [issues page](https://github.com/XRichardroidX/Add_New_Items/issues).
 
 ## Show your support
```

Comment:
Great job @XRichardroidX 🤝

- [ ] [Optional] Kindly fix the broken MIT license link to make your readme file more professional.